config/seed.js

```diff
@@ -1,44 +1,94 @@
-var db = require("../models");
-
-db.sequelize
-  .sync({force: true})
-  .then(function() {
-    createUser();
-    createAdminUser();
-  });
-
-function createUser() {
-  db.User
-    .create({
-      email: 'john@smith.com',
-      firstName: 'John',
-      lastName: 'Smith',
-      nickname: 'Johnny',
-      shortDesc: 'This is a short description',
-      description: 'This is a description',
-      phone: '+12345678'
-    })
-    .then(function(user) {
-      user1 = user;
-      db.Profile
-        .create({
-          UserId: user.id,
-          profileType: 'DriverProfile',
-          data: {
-            languagesSpoken: ['English', 'Khmer'],
-            languagesWritten: ['Khmer'],
-            drivingExperience: 10,
-            cities: ['Siem Reap', 'Phnom Penh'],
-            vehicles: ['Tuk Tuk', 'Car', 'Mini Bus']
-          }
-        });
-    });
-}
-
-function createAdminUser() {
-  db.AdminUser
-    .create({
-      email: 'admin',
-      password: 'admin'
-    })
-}
+const
+  db    = require('../models'),
+  async = require('async'),
+  _     = require('underscore'),
+  Q     = require('q'),
+  faker = require('faker');
+
+var Seed = function(){};
+
+Seed.prototype = _.extend(Seed.prototype, {
+  run: function() {
+    var _this = this;
+
+    _this.createUsers      = _.range(20).map(n => _this.createUser);
```

Comment:
'arrow function syntax (=>)' is only available in ES6 (use esnext option).<br>'n' is defined but never used.